config/api_config.go

```diff
@@ -78,3 +78,8 @@ func (cfg *APIConfig) Load(v *viper.Viper) error {
 	cfg.readIn(v)
 	return v.Unmarshal(&cfg)
 }
+
+// APIEndpoint returns the full URL for a given endpoint
+func APIEndpoint(endpoint string) string {
```

Comment:
I wanted to be able to use the "ping" default endpoint without having to create a new API config and dealt with the error. Let me know if this is the wrong way to go about this.